### PR TITLE
Update display annotation and fix bugs

### PR DESCRIPTION
**Pull request by LakshanSS**
## Purpose
- Fix https://github.com/wso2-enterprise/choreo/issues/4534
- Fix https://github.com/wso2-enterprise/choreo/issues/4455
- Fix https://github.com/wso2-enterprise/choreo/issues/4663
- Fix https://github.com/wso2-enterprise/choreo/issues/4662

## Goals
Fix display annotations and doc comments
Fix updateMessage record type issue

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

## Test environment
Ballerina SLBeta1

### CODE REVIEW

File: slack/endpoint.bal [hunk 1]
```diff
@@ -164,24 +165,24 @@ public client class Client {
     # + startOfTimeRange - Start of time range as epoch
     # + endOfTimeRange - End of time range as epoch
     # + return - Stream of MessageInfo if it is a success or an error if it is a failure
-    @display {label: "Get message history of a channel"}
-    remote isolated function getConversationHistory(@display {label: "Channel name"} string channelName, 
-                                                    @display {label: "Start of time range"} string? startOfTimeRange = (), 
-                                                    @display {label: "End of time range"}string? endOfTimeRange = ()) 
+    @display {label: "Get Messages"}
+    remote isolated function getConversationHistory(@display {label: "Channel Mame"} string channelName,
```
Comment (abeykoon) [inline slack/endpoint.bal:169]: `Channel Name` ?

File: slack/endpoint.bal [hunk 2]
```diff
@@ -306,15 +307,15 @@ public client class Client {
...
     # + return - An error if it is a failure or the 'FilesList' record if it is a success
-    @display {label: "List files"}
-    remote isolated function listFiles(@display {label: "Channel name"} string? channelName = (), 
-                                       @display {label: "Number of items per page"} int? count = (), 
-                                       @display {label: "Timestamp from"} string? tsFrom = (), 
-                                       @display {label: "Timestamp to "} string? tsTo = (), 
+    @display {label: "List Files"}
+    remote isolated function listFiles(@display {label: "Channel Name"} string? channelName = (), 
+                                       @display {label: "Items Per Page"} int? count = (), 
+                                       @display {label: "Timestamp From"} string? tsFrom = (),
```
Comment (abeykoon) [inline slack/endpoint.bal:315]: `Created After` and `Created Before` ?

File: slack/types.bal [hunk 3]
```diff
@@ -212,6 +212,27 @@ public type Message record {
...
+# 
+# + channelName - Channel name containing the message to be updated
+# + text - Text message
+# + ts - Timestamp of the message to be updated
+# + attachments - A JSON-based array of structured attachments
+# + blocks - A JSON-based array of structured blocks
+# + linkNames - Set to `true` to find and link channel names and usernames
+# + parse - Refers how messages should be treated
+# + replyBroadcast - Broadcast an existing thread reply to make it visible to everyone in the channel
+public type UpdateMessage record {
```
Comment (abeykoon) [inline slack/types.bal:225]: Pls add display label.

File: slack/endpoint.bal [hunk 4]
```diff
@@ -164,24 +165,24 @@ public client class Client {
     # + startOfTimeRange - Start of time range as epoch
     # + endOfTimeRange - End of time range as epoch
     # + return - Stream of MessageInfo if it is a success or an error if it is a failure
-    @display {label: "Get message history of a channel"}
-    remote isolated function getConversationHistory(@display {label: "Channel name"} string channelName, 
-                                                    @display {label: "Start of time range"} string? startOfTimeRange = (), 
-                                                    @display {label: "End of time range"}string? endOfTimeRange = ()) 
+    @display {label: "Get Messages"}
+    remote isolated function getConversationHistory(@display {label: "Channel Mame"} string channelName, 
+                                                    @display {label: "Start Of Time Range"} string? startOfTimeRange = (),
```
Comment (abeykoon) [inline slack/endpoint.bal:170]: `Starting From (Timestamp)` and `Ending At  (Timestamp)` ?

File: slack/types.bal (hunk 3 above)
Comment (LakshanSS) [reply slack/types.bal:226]: Done

File: slack/endpoint.bal (hunk 1 above)
Comment (LakshanSS) [reply slack/endpoint.bal]: Done

File: slack/endpoint.bal (hunk 2 above)
Comment (LakshanSS) [reply slack/endpoint.bal]: Done

File: slack/endpoint.bal (hunk 4 above)
Comment (LakshanSS) [reply slack/endpoint.bal]: Done

**Review by abeykoon — APPROVED** [review]

**Pull request merged**